sections/semantics-embedded-content.include

```diff
@@ -4021,31 +4021,31 @@
     its originating server. The sandbox is still useful, however, as it disables plugins and popups,
     thus reducing the risk of the user being exposed to malware and other annoyances.
 
-    <pre highlight="html">
-&lt;iframe title="Maps" sandbox="allow-same-origin allow-forms allow-scripts"
-        src="https://maps.example.com/embedded.html"&gt;&lt;/iframe&gt;
-    </pre>
+    <xmp highlight="html">
+      <iframe title="Maps" sandbox="allow-same-origin allow-forms allow-scripts"
+              src="https://maps.example.com/embedded.html"></iframe>;
+    </xmp>
 
   </div>
 
   <div class="example">
     Suppose a file A contained the following fragment:
 
-    <pre highlight="html">
-&lt;iframe title="Example iframe" sandbox="allow-same-origin allow-forms" src=B&gt;&lt;/iframe&gt;
-    </pre>
+    <xmp highlight="html">
+      <iframe title="Example iframe" sandbox="allow-same-origin allow-forms" src=B></iframe>
```

Comment:
should src attributes have quotes? (Also applies below)

Reply:
Fixed